contributors/guide/release-notes.md

```diff
@@ -32,6 +40,21 @@ For pull requests that don't need to be mentioned at release time, use the `/rel
     NONE
     ```
 
-To see how to format your release notes, view the kubernetes/kubernetes [pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. Pull Request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
+Your release note should be written in clear and straightforward sentences. Most often, users aren't familiar with the technical details of your PR, so consider what they _need to know_ when you write your release note.
+
+Some brief examples of release notes:
+
+```
+The deprecated flag --conntrack-max has been removed from kube-proxy. Users of this flag should switch to --conntrack-min and --conntrack-max-per-core instead. (#78399, @rikatz)
+
+The --export flag for the kubectl get command, deprecated since v1.14, will be removed in v1.18.
+
+Fixed a bug that prevents dry-run from being honored for the pod/eviction sub-resource. (#76969, @apelisse)
```

Comment:
Should we remove the **rendered** portions of the above notes eg. `(#78399, @rikatz)` and `(#76969, @apelisse)`? Having these in the above examples may lead to confusion and pr submitters may believe that they need to add this part manually as part of the release note. So update as follows:

```
The deprecated flag --conntrack-max has been removed from kube-proxy. Users of this flag should switch to --conntrack-min and --conntrack-max-per-core instead. 

The --export flag for the kubectl get command, deprecated since v1.14, will be removed in v1.18.

Fixed a bug that prevents dry-run from being honored for the pod/eviction sub-resource.
```

We should probably also have an example of the action required, since I've seen so many variations of those in PRs. Here's one based on [PR kubernetes/kubernetes #80676](https://github.com/kubernetes/kubernetes/pull/80676)

```
action required: kubeadm now deletes the bootstrap-kubelet.conf file after TLS bootstrap. User relying on bootstrap-kubelet.conf should switch to kubelet.conf that contains node credentials.
```